core/src/main/scala/org/ensime/core/debug/StructureConverter.scala

```diff
@@ -167,7 +167,7 @@ class StructureConverter(private val sourceMap: SourceMap) {
     import org.ensime.util.file._
     val pcLocation = sourceMap.newLineSourcePosition(frame.location).getOrElse(
       LineSourcePosition(
-        File(frame.location.sourcePath).canon,
+        File(frame.location.sourcePath).canon.toPath,
```

Comment:
we might want to avoid going via `File` here also, but that means we need a way of canonicalising `Path`, not just `File`